topo_processor/cli/geostore/list.py

```diff
@@ -0,0 +1,36 @@
+import click
+from linz_logger import LogLevel, get_log, set_level
+
+from topo_processor.geostore.invoke import invoke_lambda
+from topo_processor.util.time import time_in_ms
+
+
+@click.command()
+@click.option(
+    "-s",
+    "--survey",
+    required=False,
+    help="The survey to filter",
+)
+@click.option(
+    "-v",
+    "--verbose",
+    is_flag=True,
+    help="Use verbose to display debug logs",
+)
+def main(dataset_id: str, verbose: bool) -> None:
```

Comment:
Is a mapping from `--survey` to `dataset_id` intentional, or should one of them be renamed?

Reply:
Not intentional, I forgot to change `dataset_id` to `survey` after making the change to call the lambda with the survey `title`